Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -280,36 +280,63 @@ public extension Tensor {
     @inlinable
     @differentiable(
         wrt: self,
-        vjp: _vjpTransposed(withPermutations:) where Scalar: TensorFlowFloatingPoint)
+        vjp: _vjpTransposed(permutation:) where Scalar: TensorFlowFloatingPoint)
+    @available(*, deprecated, message: "Please use transposed(permutation:) instead.")
     func transposed(withPermutations permutations: Tensor<Int32>) -> Tensor {
-        return Raw.transpose(self, perm: permutations)
+        return transposed(permutation: permutations)
```

Comment:
```suggestion
        transposed(permutation: permutations)
```

Reply:
Thanks for the review. I am going to remove all the redundant `return` in the Basic.swift file to make it homogeneous.